docs/rest.md

```diff
@@ -247,6 +247,14 @@ Which channel to search in.
    derived by Geo IP lookup.
 
 ## /v2/snaps
+
+### Parameters:
+
+#### `select`
+
+Filter from the given selection. Currently only limiting to refreshable
+snaps is supported via the `refresh` key.
```

Comment:
I changed to `snap refresh --list` and updated the rest API so that we have a `select: refresh` on the find endpoint now.